torchvision/prototype/transforms/functional/_meta_conversion.py

```diff
@@ -4,7 +4,7 @@
 
 
 def _xywh_to_xyxy(xywh: torch.Tensor) -> torch.Tensor:
-    xyxy = xywh.clone()
+    xyxy = xywh.clone()  # Nice. Do we have tests to ensure that no transform does an in-place modification of input?
```

Comment:
We currently have very minimal test coverage for the transforms. I'll expand them when the design is more stable.

Reply:
This is not for now. It's just a note that we should eventually have a test that checks that no Transform modifies the original input that we pass. Agreed to keep test-coverage minimal at the moment until we have a stable design.